navigation/ca_hector/config/hector_mapping.yaml

```diff
@@ -0,0 +1,25 @@
+# Parameters for Hector SLAM - Mapping package
+
+# For a full documentation of the parameters used by this package, please check
+# http://wiki.ros.org/hector_mapping
+
+# Tf use
+use_tf_scan_transformation: true
+use_tf_pose_start_estimate: false
+
+# Map size / start point
+map_resolution: 0.050
+map_start_x: 0.53
```

Comment:
Where does this come from?

Reply:
Sorry, this was a leftover from a test, both start x and y should be 0.5